### PR TITLE
BDD stuff

### PR DESCRIPTION
**Pull request by skrix**
Installed and configured:
  rubocop
  rspec
  capybara
  faker
  travis-ci
  database_cleaner

### CODE REVIEW

**Comment by ailecksandr** [comment]
Stuff* :D 
Staff = workers

File: spec/rails_helper.rb [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# This file is copied to spec/ when you run 'rails generate rspec:install'
```
Comment (ailecksandr) [inline spec/rails_helper.rb]: remove comments here. They haven't nothing very informative, but make file look messy

File: spec/rails_helper.rb [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+# run twice. It is recommended that you do not name files matching this glob to
+# end with _spec.rb. You can configure this pattern with the --pattern
+# option on the command line or in ~/.rspec, .rspec or `.rspec-local`.
+#
+# The following line is provided for convenience purposes. It has the downside
+# of increasing the boot-up time by auto-requiring all files in the support
+# directory. Alternatively, in the individual `*_spec.rb` files, manually
+# require only the support files necessary.
+#
+Dir[Rails.root.join('spec/support/**/*.rb')].each { |f| require f }
```
Comment (ailecksandr) [inline spec/rails_helper.rb]: `.each(&method(:require))`

File: spec/rails_helper.rb [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+# Checks for pending migrations and applies them before tests are run.
+# If you are not using ActiveRecord, you can remove this line.
+ActiveRecord::Migration.maintain_test_schema!
+
+RSpec.configure do |config|
+  # Remove this line if you're not using ActiveRecord or ActiveRecord fixtures
+  config.fixture_path = "#{::Rails.root}/spec/fixtures"
```
Comment (ailecksandr) [inline spec/rails_helper.rb]: Do you plan to use fixtures? I advise you to use factories (`FactoryBot`) instead

File: spec/spec_helper.rb [hunk 4]
```diff
@@ -0,0 +1,96 @@
+# This file was generated by the `rails generate rspec:install` command. Conventionally, all
```
Comment (ailecksandr) [inline spec/spec_helper.rb]: same here about comments

File: spec/support/database_cleaner.rb [hunk 5]
```diff
@@ -0,0 +1,44 @@
+RSpec.configure do |config|
+  config.use_transactional_fixtures = false
+
+  config.before(:suite) do
+    if config.use_transactional_fixtures?
```
Comment (ailecksandr) [inline spec/support/database_cleaner.rb]: it seems like extra `if` block copied from somewhere ))

File: spec/support/database_cleaner.rb [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+  config.before(:each, type: :feature) do
+    # :rack_test driver's Rack app under test shares database connection
+    # with the specs, so continue to use transaction strategy for speed.
+    driver_shares_db_connection_with_specs = Capybara.current_driver == :rack_test
+
+    unless driver_shares_db_connection_with_specs
+      # Driver is probably for an external browser with an app
+      # under test that does *not* share a database connection with the
+      # specs, so use truncation strategy.
+      DatabaseCleaner.strategy = :truncation
```
Comment (ailecksandr) [inline spec/support/database_cleaner.rb]: In the context of this project you'll need `truncation`, remove assignation and `unless`

File: spec/support/database_cleaner.rb [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+    end
+  end
+
+  config.before(:each) do
```
Comment (ailecksandr) [inline spec/support/database_cleaner.rb]: can be presented as one-liner

File: spec/support/database_cleaner.rb [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+    DatabaseCleaner.start
+  end
+
+  config.append_after(:each) do
```
Comment (ailecksandr) [inline spec/support/database_cleaner.rb]: why not just `config.after` ?

File: spec/rails_helper.rb (hunk 3 above)
Comment (skrix) [reply spec/rails_helper.rb]: If I am using FactoryBot, then I should delete the line with ```config.fixture_path = "#{::Rails.root}/spec/fixtures"```

File: spec/support/database_cleaner.rb (hunk 8 above)
Comment (skrix) [reply spec/support/database_cleaner.rb]: Really, don`t know it before but it's the same things:
[source](https://github.com/rails/rails/blob/a74041d483041345454e766c506dab1b567dec0d/actionpack/lib/abstract_controller/callbacks.rb#L219-L220)
```
        # *_action is the same as append_*_action
        alias_method :"append_#{callback}_action", :"#{callback}_action"
```

**Pull request merged**